src/Validation/RulesGatherer.php

```diff
@@ -117,6 +124,16 @@ public function extractRulesAndMessages(ProvidesRules $providesRules, array $arg
 
         $messages = $providesRules->messages();
         $this->messages += $this->wrap($messages, $argumentPath);
+
+        $attributes = $providesRules->attributes();
+        $inputToAttributes = isset($rules[0])
+            // We might be passed just the attribute for a single field
+            // in this case, we just add it to the list.
+            ? $attributes
```

Comment:
I think we might still need to add the path. Can you add a test with `@rules` on an input field?

Reply:
You are right, in this case, I do not know how to detect if I need to use the `wrap` method or not because if I use the `@rules` directive I ended up with something like this.

```php
[
	'bar.bar' => 'baz',
]
```